_posts/2023-12-04-5.8.0-released.md

```diff
@@ -0,0 +1,126 @@
+
+### Table of Contents
+
+- [Minetest no longer ships with a default game](#minetest-no-longer-ships-with-a-default-game)
+- [New Settings GUI](#new-settings-gui)
+- [Improved Android controls](#improved-android-controls)
+- [Rewritten sound support](#rewritten-sound-support)
+- [...and more](#and-more)
+- [Conclusion](#conclusion)
+
+
+## Minetest no longer ships with a default game
+
+Minetest is not a game - it's a game creation platform that allows you to make
+your own blocky gaming experiences. Developers can
+[make](https://rubenwardy.com/minetest_modding_book/) and share games. Players
+can [find](https://content.minetest.net/) and play games using the Minetest
+client.
+
+For a long time, Minetest shipped with a default game called
+[Minetest Game](https://content.minetest.net/packages/Minetest/minetest_game/).
+This game was intentionally left bare to encourage players to mod the game and
+try out other games for Minetest. However, this led to new users thinking that
+Minetest Game was all Minetest had to offer.
+
+From 5.8.0, users will instead be directed to choose a game when first starting
+Minetest. Minetest Game will still be available to download, but the user will
+see other games as well.
+
+We'll be improving the new user experience in the future, based on feedback. For
+example, we've already started a ContentDB dialog ("Browse online content")
+redesign to make it easier to find and choose packages.
+
+{% include figure_gallery.html items=page.mtg_debundle %}
+
+
+## New Settings GUI
+
+Minetest 5.8.0 comes with a new settings menu, designed to make it easier for
+new users to find and change settings.
+
+Don't worry about the appearance - we're going to be giving the entire main
+menu a makeover in a future release.
+
+{% include figure.html src="/static/blog/5.8.0/settings.png" caption="The new settings menu" %}
+
+
+## Improved Android controls
+
+You can now place nodes using a single tap on Android. This makes building much
+easier.
+
+The Android joystick used to emulate key presses but is now read directly.
+This will make movement more responsive to user input, allowing the player to
+move in any direction and at varying speeds.
+
+
+## Rewritten sound support
+
+Minetest's sound code has been completely rewritten. This improves performance,
+fixes bugs, and unlocks new capabilities.
+
+Sounds are now lazy-loaded - this reduces memory usage and load times by only
+loading long sounds when needed.
+
+Game developers and modders have more options on how to play sound, including
+changing the start time and offsets.
+
+
+## ...and more
+
+The anti-aliasing code has been improved and modernised. There's now the option
+to choose between two new algorithms - FXAA and SSAA. FXAA is faster but is
+less accurate. SSAA is very accurate but consumes more resources.
+
+{% include figure.html src="/static/blog/5.8.0/antialiasing.webp" caption="A magnified screenshot showing how anti-aliasing smoothes pixels (rollerozxa)" %}
```

Comment:
```suggestion
{% include figure.html src="/static/blog/5.8.0/antialiasing.webp" caption="A magnified screenshot showing how anti-aliasing smoothens pixels (rollerozxa)" %}
```

Reply:
"smoothens" doesn't sound correct

Reply:
ahhh I meant smooths. Weird how that wasn't picked up by the spell checker